main/forms.py

```diff
@@ -15,21 +18,22 @@ def start_end_validation(form, field):
             'Start of activity must be earlier than end of activity')
 
 
-class AddServiceActivityForm(Form):
-    specialist = QuerySelectField(
-        query_factory=Specialist.query.all,
-        get_pk=lambda a: a.id,
-        get_label=lambda a: a.name)
-    customer = QuerySelectField(
-        query_factory=Customer.query.all,
-        get_pk=lambda a: a.id,
-        get_label=lambda a: a.name)
-    service = QuerySelectField(
-        query_factory=Service.query.all,
-        get_pk=lambda a: a.id,
-        get_label=lambda a: a.title)
-    description = StringField('Description', validators=[optional()])
-    start = DateTimeField('Start', format='%Y-%d-%m %H:%M:%S',
-                          validators=[start_end_validation])
-    end = DateTimeField('End', format='%Y-%d-%m %H:%M:%S',
-                        validators=[optional()])
+def id_exist(form, field):
+    model = eval(field.label.text)
```

Comment:
Better create a function, that accepts class name and returns validator, like:
```
def validate_id_for(model):
    def validate_id(id_value):
        model.query.filter....

validators = [DataRequired(), validate_id_for(Specialist)]
````